Content.Shared/_Sunrise/PrinterDoc/PrinterDocBoundUserInterfaceState.cs

```diff
@@ -11,16 +12,46 @@ public sealed class PrinterDocBoundUserInterfaceState : BoundUserInterfaceState
     public List<string> Templates { get; }
     public bool CanCopy { get; }
 
+    public PrinterJobView? CurrentJob { get; }
+    public List<PrinterJobView> Queue { get; }
+
     public PrinterDocBoundUserInterfaceState(
         int paperCount,
         float inkAmount,
         List<string> templates,
-        bool canCopy)
+        bool canCopy,
+        PrinterJobView? currentJob = null,
+        List<PrinterJobView>? queue = null)
     {
         PaperCount = paperCount;
         InkAmount = inkAmount;
         Templates = templates;
         CanCopy = canCopy;
+        CurrentJob = currentJob;
+        Queue = queue ?? new();
+    }
+}
+
+[Serializable, NetSerializable]
+public sealed class PrinterJobView
+{
+    public string Title { get; }
+    public PrinterJobType Type { get; }
+
+    public PrinterJobView(string title, PrinterJobType type)
+    {
+        Title = title;
+        Type = type;
+    }
```

Comment:
```suggestion
public sealed class PrinterJobView(string title, PrinterJobType type)
{
    public readonly string Title = title;
    public readonly  PrinterJobType Type = type;
```

Есть укороченный вид. А еще использовать гет сет не вписывается в кодбазу. В сс14 это используют, только если наследование напрямую используется.